packages/astro-ghostcms/src/astro-ghostcms.ts

```diff
@@ -0,0 +1,327 @@
+ */
+export default defineIntegration({
+	name: "@matthiesenxyz/astro-ghostcms",
+	optionsSchema: GhostUserConfigSchema,
+	plugins: [...corePlugins],
+	setup({ options, name }) {
+		const { resolve } = createResolver(import.meta.url);
+
+		return {
+			"astro:config:setup": ({
+				watchIntegration,
+				hasIntegration,
+				addIntegration,
+				addVirtualImports,
+				addDts,
+				injectRoute,
+				logger,
+			}) => {
+				// Configure Loggers
+				const GhostLogger = logger.fork(c.bold(c.blue("👻 Astro-GhostCMS")));
+				const GhostENVLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.blue(
+						"ENV Check",
+					)}`,
+				);
+				const GhostIntegrationLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.blue(
+						"Integrations",
+					)}`,
+				);
+				const GhostRouteLogger = logger.fork(
+					`${c.bold(c.blue("👻 Astro-GhostCMS"))}${c.gray("/")}${c.blue(
+						"Router",
+					)}`,
+				);
+
+				// Setup Watch Integration for Hot Reload during DEV
+				watchIntegration(resolve());
+				GhostLogger.info("Initializing @matthiesenxyz/astro-ghostcms...");
+
+				// Set up verbose logging
+				const verbose = options.fullConsoleLogs;
+
+				// Check for GhostCMS environment variables
+				GhostENVLogger.info(
+					c.bold(
+						c.yellow(
+							"Checking for GhostCMS environment variables & user configuration",
+						),
+					),
+				);
+
+				if (ENV.CONTENT_API_KEY === undefined) {
+					GhostENVLogger.error(
+						c.bgRed(
+							c.bold(
+								c.white("CONTENT_API_KEY is not set in environment variables"),
+							),
+						),
+					);
+					throw new AstroError(
+						`${name} CONTENT_API_KEY is not set in environment variables`,
+					);
+				}
+
+				if (options.ghostURL === undefined) {
+					GhostENVLogger.warn(
+						c.bgYellow(
+							c.bold(
+								c.black(
+									"ghostURL is not set in user configuration falling back to environment variable",
+								),
+							),
+						),
+					);
+					if (ENV.CONTENT_API_URL === undefined) {
+						GhostENVLogger.error(
+							c.bgRed(
+								c.bold(
+									c.white(
+										"CONTENT_API_URL is not set in environment variables",
+									),
+								),
+							),
+						);
+						throw new AstroError(
+							`${name} CONTENT_API_URL is not set in environment variables`,
+						);
+					}
+				}
+				GhostENVLogger.info(
+					c.bold(c.green("GhostCMS environment variables are set")),
+				);
+
+				// Set up Astro-GhostCMS Integrations
+				GhostIntegrationLogger.info(
+					c.bold(c.magenta("Configuring Enabled Integrations")),
+				);
+
+				// Theme Provider
+				if (!options.ThemeProvider?.disableThemeProvider) {
+					addIntegration(
+						ghostThemeProvider({
+							theme: options.ThemeProvider?.theme,
+							verbose,
+						}),
+					);
+				} else {
+					if (verbose) {
+						GhostIntegrationLogger.info(c.gray("Theme Provider is disabled"));
+					}
+				}
```

Comment:
```suggestion
				} else if (verbose) {
					GhostIntegrationLogger.info(c.gray("Theme Provider is disabled"));
				}
```

Reply:
I ALWAYS FORGET YOU CAN DO `else if` vscode should complain if you do an `else { if()` lol

Reply:
i think if you have eslint you can enable such a rule